src/kOS/Suffixed/WaypointValue.cs

```diff
@@ -8,14 +8,14 @@ namespace kOS.Suffixed
 {
```

Comment:
This is preferable to your previous practice of turning things private that I meant to be public - that annoyed me to no end.  I left them open so it's possible to make further derivatives of this later for other kinds of waypoints if we want to expand this.